lib/job.js

```diff
@@ -231,17 +231,19 @@ class Job extends BaseModel {
 
         try {
             const newPeriodic = getAnnotations(newJob.permutations[0], 'screwdriver.cd/buildPeriodically');
-            const isJobEnabled = newJob.state === 'ENABLED';
-            const isJobArchived = newJob.archived;
+            const newIsJobEnabled = newJob.state === 'ENABLED';
```

Comment:
`newIsJobEnabled ` => `isNewJobEnabled`?

Reply:
Thank you. I fixed.